staging/src/k8s.io/controller-manager/app/serve.go

```diff
@@ -18,6 +18,7 @@ package app
 
 import (
 	"net/http"
+	"runtime"
```

Comment:
this is already imported as `goruntime` in the line below, we should not import it under two names

it is probably imported as `goruntime` to disambiguate from the api machinery runtime package